src/rantanplan/core.py

```diff
@@ -72,8 +72,11 @@
 # words starting with OBREP-
 OBREP_RE = re.compile("^(ob)(rep.*)", re.I | re.U)
 
-# words starting with prefixes SIN-/DES- followed by consonant "destituir", "sinhueso"
-PREFIXES_WITH_CONSONANT_RE = (re.compile("^(des|sin)([bcdfgjklmhnñpqrstvxyz].*)", re.I | re.U))
+# words starting with prefixes SIN-/DES- followed by consonant "destituir"
+PREFIX_DES_WITH_CONSONANT_RE = (re.compile("^(des|sin)([bcdfgjklmhnñpqrstvxyz].*)", re.I | re.U))
+
+# words starting with prefixes SIN-/DES- followed by consonant "sinhueso"
+PREFIX_SIN_WITH_CONSONANT_RE = (re.compile("^(des|sin)([bcdfgjklmhnñpqrstvxyz].*)", re.I | re.U))
```

Comment:
If the regular expression is exactly the same, why split it in two variables of identical content?

Reply:
Yes, they'll have to be different for future work, changing it now